Add a FAQ to the README

davidkrauser opened the pull request:
This PR introduces a quick FAQ section to the README.

davidkrauser commented on README.md:
```diff
@@ -43,8 +43,31 @@ The Azure AI Content Safety API uses severity levels from 0-6:
+
+### How can I monitor moderation activity?
+
+Currently, moderation events are logged in the Mattermost server logs. Future versions will include metrics visualization support for better monitoring and reporting.
```
We don't have audit log support, yet. Still waiting on server support: https://github.com/mattermost/mattermost/pull/31204

wiggin77 commented on README.md:
```diff
@@ -43,8 +43,31 @@ The Azure AI Content Safety API uses severity levels from 0-6:
+
+### What if content moderation APIs are unavailable?
+
+The plugin uses a "fail-open" approach for reliability. If the moderation API is unavailable or returns an error, no posts are moderated.
```
Is there something specific in logs that can be used to monitor for unavailability?

wiggin77 replied:
> Yes there is - they'll see a log statement with this error: `"Content moderation error: moderation service is not available"`

Great. Please add that to the faq.

wiggin77 commented on README.md:
```diff
@@ -43,8 +43,31 @@ The Azure AI Content Safety API uses severity levels from 0-6:
+
+### Can I exclude certain users from moderation?
+
+Yes, you can specify user IDs in the "Excluded Users" configuration setting. All other users will have their content moderated automatically.
```
Does this support user groups?

davidkrauser replied:
Yes it does, but the README doesn't mention it - forgot to add that. I'll make that change :+1:

wiggin77 replied:
> No - but we do support excluding by channel. The README doesn't mention it - forgot to add that. This is now included below :+1: 

Ok, let's create a ticket to support groups in future.

davidkrauser commented:
@wiggin77 are you able to take another look at this? I've answered your questions and updated the README to include documentation around excluding by channel.

Also fixed a minor issue with C/I that was introduced with the upgrade to golang 1.24
